Apps/Common/src/Models/PHSA/PHSALoadState.cs

```diff
@@ -0,0 +1,46 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.Common.Models.PHSA
```

Comment:
Why is this in common?

Reply:
as we have an existing PHSA folder in Common project and this is also related to PHSA so I was thinking it was in the right place, but we can chat for the other options.

Reply:
I think is in Common since the load state is shared across the PHSA APIs. Both Immz and Lab have the same structure